fitbenchmarking/fitbenchmarking
# Added synthetic Gaussian data to test mantid fitting

## Description of work

Fixes #1359 

Added Richard's low stats gaussian data with varying starting values for the fit-able parameters. The summary of how the parameters are varied is shown below.  

Dataset | Filename | Height | Peak Centre | Sigma | A0 | 
--- | --- | --- | --- | --- |--- |
1 | `Height-1` | $\color{#0437F2}\textsf{\normalsize 8}$ | 7  | 0.2 | 0 |
2 | `Height-2` | $\color{#0437F2}\textsf{\normalsize 10}$ | 7 | 0.2 | 0 |
3 | `Height-3` | $\color{#0437F2}\textsf{\normalsize 12}$ | 7 | 0.2 | 0 |
4 | `Height-4`| $\color{#0437F2}\textsf{\normalsize 14}$ | 7 | 0.2 | 0 |
5 | `Peak-Centre-1` | 7 | $\color{#0437F2}\textsf{\normalsize 6}$ | 0.2 | 0 |
6 | `Peak-Centre-2` | 7 | $\color{#0437F2}\textsf{\normalsize 6.5}$ | 0.2 | 0 |
7 | `Peak-Centre-3` | 7 | $\color{#0437F2}\textsf{\normalsize 7.5}$ | 0.2 | 0 |
8 | `Peak-Centre-4` | 7 | $\color{#0437F2}\textsf{\normalsize 8}$ | 0.2  | 0 |
9 | `Sigma-1` | 7 | 7 | $\color{#0437F2}\textsf{\normalsize 0.4}$ | 0 | 
10 | `Sigma-2` | 7 | 7 | $\color{#0437F2}\textsf{\normalsize 0.6}$ | 0 |
11 | `Sigma-3` | 7 | 7 | $\color{#0437F2}\textsf{\normalsize 0.8}$ | 0 |
12 | `Sigma-4` | 7 | 7 | $\color{#0437F2}\textsf{\normalsize 1}$ | 0 |
13 | `A0-1` | 7 | 7 | 0.2 | $\color{#0437F2}\textsf{\normalsize 1}$ |
14 | `A0-2` | 7 | 7 | 0.2 | $\color{#0437F2}\textsf{\normalsize 0.5}$ |
15 | `A0-3` | 7 | 7 | 0.2 | $\color{#0437F2}\textsf{\normalsize -0.5}$ |
16 | `A0-4` | 7 | 7 | 0.2 | $\color{#0437F2}\textsf{\normalsize -1}$ |





## Testing Instructions



1. Run `fitbenchmarking -p examples/benchmark_problems/Synthetic_Data/low_stats_gaussian -s scipy mantid gofit` and verify that results are produced.
2. I found that `Powell` from **scipy** was doing almost as well as the `Multistart` from **gofit**. But `Simplex` from **mantid** was also among the top 3.
<img width="495" alt="image" src="https://github.com/user-attachments/assets/c67a58ad-11af-4785-af16-77c7dab221ab">



## Checklist
